tests/test_jenkins.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+
```

Comment:
Usually there isn't an extra space between the function signature and the function comment.